### PR TITLE
Issue #383 ability to exclude images

### PR DESCRIPTION
**Pull request by jammasterj89**
@konstruktoid as per our discussion on issue #383

### CODE REVIEW

File: docker-bench-security.sh [hunk 1]
```diff
@@ -103,15 +101,27 @@ main () {
       benchcont="$c"
     fi
   done
+
```
Comment (konstruktoid) [inline docker-bench-security.sh]: White spaces.

**Comment by konstruktoid** [comment]
LGTM but remove the white spaces and please sign your commits.
Closes #383

**Comment by konstruktoid** [comment]
Closes  #369

File: docker-bench-security.sh (hunk 1 above)
Comment (jammasterj89) [reply docker-bench-security.sh]: Fixed in 3d02432bc8255c3b115ba335d0fe4c3e24a8cd37

**Comment by konstruktoid** [comment]
Thanks!

**Comment by kumarchatla** [comment]
is -i and -x both should include to run on specific images? I have 100 images and I want to scan only one image in that. Is it possible with -i?

**Comment by konstruktoid** [comment]
Examples (I've removed the yell):
```
$ docker images
REPOSITORY                  TAG                 IMAGE ID            CREATED             SIZE
konstruktoid/nginx          latest              b05b335b68d0        23 hours ago        10.6MB
konstruktoid/docker-garby   latest              57fdebbb0bb1        23 hours ago        207MB
nginx                       latest              231d40e811cd        10 days ago         126MB
konstruktoid/alpine         latest              c4e0b1901367        5 weeks ago         5.57MB
alpine                      latest              965ea09ff2eb        6 weeks ago         5.55MB
$ sudo sh docker-bench-security.sh -c check_4_6
[WARN] 4.6  - Ensure that HEALTHCHECK instructions have been added to container images
[WARN]      * No Healthcheck found: [konstruktoid/nginx:latest]
[WARN]      * No Healthcheck found: [nginx:latest]
[WARN]      * No Healthcheck found: [konstruktoid/alpine:latest]
[WARN]      * No Healthcheck found: [alpine:latest]

[INFO] Checks: 1
[INFO] Score: -1
$ sudo sh docker-bench-security.sh -c check_4_6 -i nginx
[WARN] 4.6  - Ensure that HEALTHCHECK instructions have been added to container images
[WARN]      * No Healthcheck found: [nginx:latest]

[INFO] Checks: 1
[INFO] Score: -1
$ sudo sh docker-bench-security.sh -c check_4_6 -x nginx
[WARN] 4.6  - Ensure that HEALTHCHECK instructions have been added to container images
[WARN]      * No Healthcheck found: [konstruktoid/alpine:latest]
[WARN]      * No Healthcheck found: [alpine:latest]

[INFO] Checks: 1
[INFO] Score: -1
```

**Comment by kumarchatla** [comment]
I'm not sure how it is working for you. I'm tr
workbench_without i
workbench_with i

ying the same way. Still No luck :(

**Comment by nialltu** [comment]
@kumarchatla that is the correct behaviour. 

You are running one test against one image with no failures, typically the failed image is shown beneath the failed test e.g. in your case which container does not have a healthcheck. However, that test has passed so that looks correct.

**Comment by jammasterj89** [comment]
@kumarchatla that behaviour is correct.

You are running one test against one container and it has passed. Typically, if it fails it will indicate which container has failed the test. However, in your case there are no failures so this is expected.

**Comment by kumarchatla** [comment]
If I don't use **-i**, I am able to see failed for the same image ( Running against all images). If I include ( -i imagename) then that image was passing with no health check complaints.

**Comment by jammasterj89** [comment]
Apologies, I misread that. I'll try take a look when I get some time.

**Comment by konstruktoid** [comment]
@kumarchatla that's odd, just to be sure. could you use `-i mbt` ?

**Comment by jammasterj89** [comment]
This looks to be only an issue with the images (not containers) because of the stripping of the top row (sed '1d') before searching the results. I've reordered this so the sed command comes first, it should be fixed it in PR #406.

**Comment by kumarchatla** [comment]
> This looks to be only an issue with the images (not containers) because of the stripping of the top row (sed '1d') before searching the results. I've reordered this so the sed command comes first, it should be fixed it in PR #406.

This Works, Thanks

**Pull request merged**